configure.ac

```diff
@@ -2997,6 +2997,10 @@ int main(int argc,char **argv)
   found_iconv=no
 ,
   found_iconv="$cross_found_iconv_libc",
+  if test "$found_iconv" = "yes"; then
```

Comment:
I see no lines in configure that are caused by this change. Did you forget to regenerate configure or is this change not needed?

Reply:
Good catch! This was caused by the line above incorrectly ending in a comma instead of a semicolon.